specification/compute/resource-manager/Microsoft.Compute/ComputeRP/stable/2022-11-01/virtualMachine.json

```diff
@@ -1160,6 +1174,20 @@
             "required": false,
             "type": "string",
             "description": "The system query option to filter VMs returned in the response. Allowed value is 'virtualMachineScaleSet/id' eq /subscriptions/{subId}/resourceGroups/{resourceGroupName}/providers/Microsoft.Compute/virtualMachineScaleSets/{vmssName}'"
+          },
+          {
+            "name": "$expand",
+            "in": "query",
+            "required": false,
+            "type": "string",
+            "description": "The expand expression to apply on operation. 'instanceView' enables fetching run time status of all Virtual Machines, this can only be specified if a valid $filter option is specified",
```

Comment:
lgtm
Maybe mention in the description or through examples the expected result if the param is provided with an invalid `$filter` option